docs/tables/aws_config_conformance_pack.md

```diff
@@ -7,17 +7,41 @@ A conformance pack is a collection of AWS Config rules and remediation actions t
 ### Basic info
 
 ```sql
-select 
-    name, 
-    conformance_pack_id, 
-    input_parameters, 
-    created_by_svc, 
-    delivery_bucket, 
-    delivery_bucket_prefix, 
-    last_update, 
-    title, 
-    akas
-from 
-    aws.aws_config_conformance_pack;
+select
+  name,
+  conformance_pack_id,
+  created_by,
+  last_update_requested_time,
+  title,
+  akas
+from
+  aws_config_conformance_pack;
 ```
 
+
+### Input parameter details of each conformance pack
+
+```sql
+select
```

Comment:
What type of information is contained in the input parameters field?

Reply:
It contains Input parameters in the form of key-value pairs for the conformance pack -`ParameterName`  and `ParameterValue`